R/locations.R

```diff
@@ -68,7 +74,16 @@ aq_locations <- function(country = NULL, city = NULL, location = NULL,# nolint
                          latitude = latitude,
                          longitude = longitude,
                          radius = radius,
-                         page = page)
+                         page = page,
+                         siteType = siteType,
+                         completeness_from = completeness_from,
+                         completeness_to = completeness_to,
+                         activation_date_from = activation_date_from,
+                         activation_date_to = activation_date_to,
+                         inletHeight_from = inletHeight_from,
+                         inletHeight_to = inletHeight_to)
+
+  return(argsList)
```

Comment:
```suggestion
```

You forgot to remove this :wink:

Reply:
oh yes of course 🙈

Reply:
Sorry for reaching out again... I didn't have any time to look into this. Probably this is solved by now anyways.

Reply:
actually no :joy: so if you have time to work on this feel free to ping me. 
I need to do a release of ropenaq next week, but this feature could go to the release after that.